Add title box to `Picker` (#8155)
helix-editor/helix: book/src/configuration.md

| `smart-case` | Enable smart case regex searching (case-insensitive unless pattern contains upper case characters) | `true` |
| `wrap-around`| Whether the search should wrap after depleting the matches | `true` |

### `[editor.picker]` Section

Picker specific options.

| Key | Description | Default |
|--|--|---------|

> | `picker-title` | Renders a title for the opening picker. Can be `never`, `center`, `inline`, `inline-border` or `prompt` | `never` |

Reviewer: The formulation here can make a non-fluent reader think that the title will be `never`, `center`, ...

What do you think about:

```suggestion
| `title-format` | How the picker's title is displayed. Can be `never`, `center`, `inline`, `inline-border` or `prompt` | `never` |
```
